crates/zed/src/languages/clojure/config.toml

```
@@ -0,0 +1,10 @@
+name = "Clojure"
+path_suffixes = ["clj", "cljs"]
+line_comment = "# "
```

```suggestion
line_comments = ["# "]
```
We've changed the config for `line_comments` to array recently.